### ISSUE TITLE
What are the returns from fit_im?

### ISSUE THREAD

**Issue by keflavich** [schlafly/crowdsource#9]
Could you provide some narrative documentation for the returns from fit_im?

https://github.com/schlafly/crowdsource/blob/d0bb2ebfa3d57611079581b4601f9056efa0c615/crowdsource/crowdsource_base.py#L695-L696

My attempt:
 * `stars` is the star catalog
 * `model + sky` is the sum of the stellar model and the "background" model
 * `sky + msky` is the .... sky model?  It's low-resolution.  I'm not sure how to differentiate `sky`, `msky`, and `model`, though I suspect `model` is the stars-only image?
 * `psf` is the fitted PSF (or the original PSF if refitting isn't used)

Which return value would you consider the best representation of the star-subtracted image?  i.e., the best representation of the extended emission?

**Comment by schlafly** [comment]
Thanks for looking at this.
Yes:
- stars is the star catalog
- model + sky is the total model; the sum of the model and the sky
- sky + msky is the total diffuse component; it's the sum of the model sky (msky; already included in model), and the high frequency median sky (sky) that I fit separately from the model.
- psf is the psf model.
The best representation of the star-subtracted image would be sky+msky; that's what that third entry is intended to be.

**Comment by keflavich** [comment]
Great, thanks!

Shall we leave this open until you get a chance to put this into the documentation somewhere?

**Comment by schlafly** [comment]
Yes, I'll leave it open as a reminder that I should add some docstrings for functions---certainly at least some of the major entrance points.

**Comment by andrew-saydjari** [comment]
I will say that I also had to explain this to someone in the last 3 months, so it is a high priority doc string.
